parlai/agents/transformer/biencoder.py

```diff
@@ -107,3 +107,9 @@ def _set_text_vec(self, *args, **kwargs):
             )
             obs['added_start_end_tokens'] = True
         return obs
+
+
+class IRFriendlyBiencoderAgent(AddLabelFixedCandsTRA):
```

Comment:
Then wouldn't this need `IRFriendlyBiencoderAgent(AddLabelFixedCandsTRA, BiEncoder)`?

Reply:
🤦‍♂